packages/default-plugins/build.ts

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable no-console */
+
+import { copy, exists, remove, mkdir, readdir, mkdtemp } from 'fs-extra';
+import { dirname, join, resolve, basename } from 'path';
+import { tmpdir } from 'os';
+import { chdir, cwd } from 'process';
+import { execCommand } from '@joplin/utils';
```

Comment:
**To-do**: Using `@joplin/utils` during the first build fails if `@joplin/utils` hasn't been built yet.

Reply:
Is it added to package.json?

Reply:
Yes &mdash; however, the default plugins build script is currently run during the build process, which seems to sometimes happen before `tsc` compiles `@joplin/utils`.